UefiTestingPkg/AuditTests/PagingAudit/Windows/BinaryParsing.py

```diff
@@ -65,17 +65,17 @@ def Parse4kPages(fileName, addressbits, architecture):
             if 0 == (ByteArray[byteZeroIndex + 0] + ByteArray[byteZeroIndex + 1] + ByteArray[byteZeroIndex + 2] + ByteArray[byteZeroIndex + 3] + ByteArray[byteZeroIndex + 4] + ByteArray[byteZeroIndex + 5] + ByteArray[byteZeroIndex + 6] + ByteArray[byteZeroIndex + 7]):
                 byteZeroIndex += 8
                 continue
-            Valid = ((ByteArray[byteZeroIndex + 0] & 0x1))
+            AccessFlag = ((ByteArray[byteZeroIndex + 1] & 0x4) >> 2)

```

Comment:
Like the other file, do you know why constants like `0x4` and `2` are not defined in architectural header files?

Reply:
Is there a way to include C header files with Python scripts? Or do you mean there should be some universal header file?

Reply:
Mispoke on it as a C file looking at the diff quickly. Python would provide better abstractions with its object model, but if you're not aware of any (public) libraries that readily provide hardware access abstractions then please ignore this comment.

Reply:
I am not aware of any and want to avoid refactoring the pre-existing parsing conventions in the script as I don't think it's very valuable work